mobile/src/navigation/app/common/constants/constants.ts

```diff
@@ -111,20 +114,29 @@ const NAVIGATION_ITEMS: DrawerNavigationItem[] = [
     permissions: [PermissionKey.MANAGE_UAM],
     isAuthRequired: true,
     drawerGroup: 'Account',
+    screenOptions: {
+      swipeEdgeWidth: 10,
+    },
   },
   {
     name: AppScreenName.UAM_GROUPS_CREATE,
     icon: 'uam',
     component: UAMConfigureGroup,
     permissions: [PermissionKey.MANAGE_UAM],
     isAuthRequired: true,
+    screenOptions: {
+      swipeEnabled: false,
```

Comment:
Let's have same behavior for screens with tables
```suggestion
      swipeEdgeWidth: 10,
```